core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationWarnings.java

```diff
@@ -61,9 +71,22 @@ public static void add(INamedObject object, Logger log, String message) {
 	/**
 	 * Adds configuration warning in case warning is not suppressed
 	 */
-	public static void add(IConfigurable object, Logger log, String message, String suppressionKey, IAdapter adapter) {
+	public static void add(IConfigurable object, Logger log, String message, SuppressKeys suppressionKey, IAdapter adapter) {
 		if(!isSuppressed(suppressionKey, adapter, object.getConfigurationClassLoader())) {
 			add(object, log, message, null);
+			// provide suppression hint as info 
+			if(log.isInfoEnabled()) {
+				if(adapter != null) {
+					String messageWithHint = "in adapter [" + adapter.getName() + "] " + message + ". This warning can be suppressed by setting the property '"+suppressionKey.getKey()+"."+adapter.getName()+"=true'";
+					if(suppressionKey.isAllowGlobalSuppression()) {
+						messageWithHint += ", or globally by setting the property '"+suppressionKey.getKey()+"=true'";
+					}
+					log.info(messageWithHint);
+				} else if(suppressionKey.isAllowGlobalSuppression()) {
+					String messageWithHint = message + "This warning can be suppressed globally by setting the property '"+suppressionKey.getKey()+"=true'";
+					log.info(messageWithHint);
```

Comment:
Now msg is logged twice: first without the hint in add(), then here with the hint.
This should not happen. The message should be logged only once, (with or without hint, depending on log.isInfoEnabled())